Source/Scene/Model.js

```diff
@@ -810,17 +819,17 @@ define([
             setCachedGltf(model, cachedGltf);
             gltfCache[cacheKey] = cachedGltf;
 
-            if (/\.bgltf$/i.test(url)) {
-                // Load binary glTF
-                loadArrayBuffer(url, options.headers).then(function(arrayBuffer) {
+            loadArrayBuffer(url, options.headers).then(function(arrayBuffer) {
+                var magic = getStringFromTypedArray(arrayBuffer, 0, 4);
```

Comment:
Can't fit any valid GLTF JSON in 3 bytes or less.  Should we explicitly throw RuntimeError?